src/NServiceBus.Metrics.PerformanceCounters.Tests/PerformanceCounterUpdaterTests.cs

```diff
@@ -84,125 +76,73 @@ public void Meters_that_need_mapping_within_payload_should_be_converted_into_per
+
+            var durationProbes = new[]
+            {
+                new MockDurationProbe("Critical Time"),
+                new MockDurationProbe("Processing Time")
+            };
+
+            sut.Observe(new ProbeContext(durationProbes, new ISignalProbe[0]));
 
-            sut.Update(PayloadWithRandomTimers);
+            durationProbes[0].Observers(TimeSpan.FromSeconds(11));
+            durationProbes[1].Observers(TimeSpan.FromSeconds(22));
 
             var performanceCounterOne = cache.Get(new CounterInstanceName("Critical Time", "Sender@af016c07"));
             var performanceCounterTwo = cache.Get(new CounterInstanceName("Processing Time", "Sender@af016c07"));
             
             Assert.AreEqual(11, performanceCounterOne.RawValue);
             Assert.AreEqual(22, performanceCounterTwo.RawValue);
         }
-
-        const string PayloadWithRandomTimers = @"{	
-    ""Context"": ""Sender@af016c07"",    
-    ""Timers"": [
-      {
-        ""Name"": ""Critical Time"",
-        ""Count"": 0,
-        ""ActiveSessions"": 0,
-        ""TotalTime"": 11,
-        ""Rate"": {
-          ""MeanRate"": 0.0,
-          ""OneMinuteRate"": 0.0,
-          ""FiveMinuteRate"": 0.0,
-          ""FifteenMinuteRate"": 0.0
-        },
-        ""Histogram"": {
-          ""LastValue"": 11000.0,
-          ""Min"": 0.0,
-          ""Mean"": 0.0,
-          ""StdDev"": 0.0,
-          ""Median"": 0.0,
-          ""Percentile75"": 0.0,
-          ""Percentile95"": 0.0,
-          ""Percentile98"": 0.0,
-          ""Percentile99"": 0.0,
-          ""Percentile999"": 0.0,
-          ""SampleSize"": 0
-        },
-        ""Unit"": ""Messages"",
-        ""RateUnit"": ""s"",
-        ""DurationUnit"": ""ms""
-      },
-      {
-        ""Name"": ""Processing Time"",
-        ""Count"": 0,
-        ""ActiveSessions"": 0,
-        ""TotalTime"": 22,
-        ""Rate"": {
-          ""MeanRate"": 0.0,
-          ""OneMinuteRate"": 0.0,
-          ""FiveMinuteRate"": 0.0,
-          ""FifteenMinuteRate"": 0.0
-        },
-        ""Histogram"": {
-          ""LastValue"": 22000.0,
-          ""Min"": 0.0,
-          ""Mean"": 0.0,
-          ""StdDev"": 0.0,
-          ""Median"": 0.0,
-          ""Percentile75"": 0.0,
-          ""Percentile95"": 0.0,
-          ""Percentile98"": 0.0,
-          ""Percentile99"": 0.0,
-          ""Percentile999"": 0.0,
-          ""SampleSize"": 0
-        },
-        ""Unit"": ""Messages"",
-        ""RateUnit"": ""s"",
-        ""DurationUnit"": ""ms""
-      }
-    ]
-  }";
     }
-
+    
     class MockPerformanceCountersCache : PerformanceCountersCache
     {
         protected override IPerformanceCounterInstance CreateInstance(CounterInstanceName counterInstanceName)
         {
             return new MockIPerformanceCounter();
         }
     }
+
+    class MockSignalProbe : ISignalProbe
```

Comment:
These mocks are so straightforward, comparing to the previous Metrics.NET approach 😍